packages/client/src/mud/createSystemCalls.ts

```diff
@@ -45,22 +48,39 @@ export function createSystemCalls(
     return getComponentValue(Counter, singletonEntity);
   };
 
-  const mudIsPositionEmpty = async (
-    position: Vector3 = new Vector3(1, 1, 1)
-  ) => {
-    const res = await worldContract.read.isPositionEmpty([
-      position.x,
-      position.y,
-      position.z,
-    ]);
+  const mudGetEntityType = async (entityKey) => {
+    return getComponentValue(EntityType, entityKey);
+  };
+
+  const mudGetOwnedBy = async (entityKey) => {
+    return getComponentValue(OwnedBy, entityKey);
+  };
+
+  const mudGetOrientation = async (entityKey) => {
+    return getComponentValue(Orientation, entityKey);
+  };
+
+  const mudGetPosition = async (entityKey) => {
+    return getComponentValue(Position, entityKey);
+  };
+
+  const mudPositionToEntityKey = (pos: Vector3) => {
```

Comment:
forgot what keccak did 🤣 my Solidity is rusty, this makes a lot of sense.

Reply:
its a hash 256 function widely used in Solidity 🤣